gridcapa-core-valid-app/src/main/java/com/farao_community/farao/gridcapa_core_valid/app/study_point/StudyPointService.java

```diff
@@ -133,6 +141,10 @@ private void resetInitialPminPmax(Network network, ZonalData<Scalable> scalableZ
         LOGGER.info("Pmax and Pmin are reset to initial values for network {}", network.getNameOrId());
     }
 
+    private RaoRequest buildRaoRequest(String requestId, String cbcora, String cgm, String glsk) {
+        return new RaoRequest(requestId, null, cgm, null, cbcora, glsk, null, null, null);
```

Comment:
It's better to use another request Id for each RaoRequest. For example
raoRequestId = String.format("%s-%s", requestId, studyPoint.getId());